src/RadhydroSimulation.hpp

```diff
@@ -98,7 +103,6 @@ template <typename problem_t> class RadhydroSimulation : public AMRSimulation<pr
 
 	bool computeReferenceSolution_ = false;
 	amrex::Real errorNorm_ = NAN;
-	amrex::Real densityFloor_ = 0.;
```

Comment:
Since the densityFloor_ already exists in RadhydroSimulation, I think it makes more sense to put the limits there. You can then read them from the input file in the constructor.